### ISSUE TITLE
jquery plugins should not be declared inside $(document).ready(...)

### ISSUE THREAD

**Issue by waynebaylor** [Dogfalo/materialize#736]
For example, the $.fn.material_select() plugin is defined within a call to $.ready(). 

I will get an error if I try to execute the following:

```
...
<body>
    ...
    <script src="js/materialize.js"></script>
    <script>
        $("select").material_select();  // <--- ERROR: material_select not found
    </script>
</body>
```

This is bad. 

All the materialize plugins should be defined once the `materialize.js` script has been parsed.

**Comment by acburst** [comment]
We'll look into moving our plugins. For now though, if you wrap your initializations within document ready's it should always work.

**Comment by Dogfalo** [comment]
Fixed in e07cf3239d327140a8ee69917cc0cc1cd86e0803
